cedar-policy-core/src/parser/cst_to_ast.rs

```diff
@@ -75,14 +79,27 @@ lazy_static::lazy_static! {
 }
 fn load_styles() -> ExtStyles<'static> {
     let mut functions = HashSet::new();
+    let mut unqualified_functions = HashSet::new();
     let mut methods = HashSet::new();
     for func in crate::extensions::Extensions::all_available().all_funcs() {
         match func.style() {
-            CallStyle::FunctionStyle => functions.insert(func.name()),
-            CallStyle::MethodStyle => methods.insert(func.name().basename()),
+            CallStyle::FunctionStyle => {
+                let name = func.name();
+                functions.insert(name);
+                if name.is_unqualified() {
+                    unqualified_functions.insert(name.basename_as_ref());
+                }
+            }
+            CallStyle::MethodStyle => {
+                methods.insert(func.name().basename());
```

Comment:
Might be nice to add an assertion here that `func.name().is_unqualified()`

Reply:
Added